docs/resources/virtual_environment_container.md

```diff
@@ -136,6 +136,17 @@ output "ubuntu_container_public_key" {
     - `dedicated` - (Optional) The dedicated memory in megabytes (defaults
       to `512`).
     - `swap` - (Optional) The swap size in megabytes (defaults to `0`).
+- `mount_point`
+    - `acl` (Boolean) Explicitly enable or disable ACL support
+    - `backup` (Boolean) Whether to include the mount point in backups (only used for volume mount points)
+    - `mount_options` (List of String) Extra mount options.
+    - `path` (Required) Path to the mount point as seen from inside the container
+    - `quota` (Boolean) Enable user quotas inside the container (not supported with zfs subvolumes)
+    - `read_only` (Boolean) Read-only mount point
+    - `replicate` (Boolean) Will include this volume to a storage replica job
+    - `shared` (Boolean) Mark this non-volume mount point as available on all nodes
+    - `size` (String) Volume size (read only value)
+    - `volume` (Required) Volume, device or directory to mount into the container
```

Comment:
TODO: document that directory mounts require that you log in with `root@pam` and a password, nothing else works